sale_order_lot_selection/__init__.py

```diff
@@ -1 +1,5 @@
+# Copyright (C) 2025 Cetmix OÜ
```

Comment:
Don't put this in __init__.py, this is not mandatory (there is no code in it :smile: )